tls/ttls.c

```diff
@@ -128,7 +128,7 @@ module_exit(ttls_exit);
 
 MODULE_AUTHOR("Tempesta Technologies");
 MODULE_VERSION("2.3.0");
-MODULE_LICENSE("GPLv2");
+MODULE_LICENSE("GPL");
```

Comment:
I have nothing against change of license but if this change is to fix tainting of kernel misspelled `GPLv2` token can be replaced with `GPL v2` without change of sense. Here is list of supported license identifiers: http://lxr.free-electrons.com/source/include/linux/module.h#L172